Readme.md

```diff
@@ -111,3 +111,20 @@ To stop, hit `CTRL-C` in the terminal where you started `runserver`, and
 The deployment is orchestrated at https://github.com/vmck/acs-deploy - it's the
 living documentation and single instance of how to run this thing in
 production.
+
+## Add new homework
+1. You need to have access to the [admin page for V2 vmchecker](http://v2.vmchecker.cs.pub.ro/admin).
```

Comment:
Please write the address with `https://`. The `http://` does indeed redirect to the https version, but it's more secure if you link directly to https.

Reply:
Done